src/common/main/kotlin/tech/thatgravyboat/skyblockapi/utils/Logger.kt

```diff
@@ -6,7 +6,7 @@ internal object Logger {
 
     fun debug(message: String, vararg args: Any) {
         if (!SkyBlockAPI.isDebug) return
-        warn("[DEBUG] $message", *args)
+        SkyBlockAPI.logger.debug(message, *args)
```

Comment:
iirc you cant actually change the log level in prod so this was just away around that